sql/08_create_taxonomic_table.sql

```diff
@@ -0,0 +1,69 @@
+-- Note: FIRST: edit %db_owner% to the account name of the DB owner 
+
+CREATE TABLE lter_metabase."TaxaList" (
+	"TaxonID" character varying(50) NOT NULL;
```

Comment:
The extra semi-colons I'm editing out and will send so you (@atn38) can re-submit via git.